src/audio/buffer.c

```diff
@@ -22,41 +22,42 @@
 /* 42544c92-8e92-4e41-b679-34519f1c1d28 */
 DECLARE_SOF_UUID("buffer", buffer_uuid, 0x42544c92, 0x8e92, 0x4e41,
 		 0xb6, 0x79, 0x34, 0x51, 0x9f, 0x1c, 0x1d, 0x28);
-uintptr_t buffer_uuid_ptr = SOF_UUID(buffer_uuid);
+DECLARE_TR_CTX(buffer_tr, SOF_UUID(buffer_uuid), LOG_LEVEL_INFO);
```

Comment:
```
/**
 * Trace context.
 */
struct tr_ctx {
	uintptr_t uuid_p;	/**< UUID pointer, use SOF_UUID() to init */
	uint32_t level;		/**< Default log level */
};
```

`LOG_LEVEL_INFO` is default log level for `buffer_tr` (default log threshold). This value may be changed in runtime, to change current trace level for each log function referencing to `buffer_tr`. So it's related more with tracing component rather than trace log message.
On the other side we have log level, related with log message. This one is defined in `tr_info`, `tr_err`... functions and it can't be change in runtime.